include/seqan/basic/alphabet_simple_type.h

```diff
@@ -319,7 +380,7 @@ struct MinValue<SimpleType<TValue, TSpec> >
 };
 
 template <typename TValue, typename TSpec>
-const SimpleType<TValue, TSpec> MinValue<SimpleType<TValue, TSpec> >::VALUE = SimpleType<TValue, TSpec>(0);
+const SimpleType<TValue, TSpec> MinValue<SimpleType<TValue, TSpec> >::VALUE = std::numeric_limits<SimpleType<TValue, TSpec>>::min();
```

Comment:
Please check whether these specializations can be remove now, also MaxValue, MinValue, infimumValueImpl(), supremumValueImpl()